docs/cvl/builtin.md

```diff
@@ -4,7 +4,84 @@ Built-in Rules
 The Prover has some built-in general-purpose rules that can be verified on a
 contract out-of-the-box.
 
+## Deep Sanity
+
+Enable with:
+```cvl
+use builtin rule deepSanity;
+```
+
+Ensure `--multi_assert_check` is enabled (will throw an error otherwise!).
+
+One can configure the number of branching nodes that will be selected (it will always pick those nodes that dominate more) by setting `--settings -maxNumberOfReachChecksBasedOnDomination=N`, default `N=10`.
+
+### Background and motivation
+
+Sanity rules are one of our main methods to setup a new code base for verification. It serves two needs:
+
+- Checking that the Prover is able to solve through a path of the code, and that there are no obvious vacuities.
+- Checking that the Prover is able to find said path in a reasonable amount of time for both pre-processing and SMT phases.
```

Comment:
replace with
2. Checking that the Prover can find the said path in a reasonable amount of time for both pre-processing and SMT phases.